Sentinel/Classes/EmailSender/EmailSenderTool.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EmailSenderTool.swift
+//  Sentinel
+//
+//  Created by Antonijo Bezmalinovic on 22.11.2023..
+//
+
+import Foundation
+import MessageUI
+
+@objcMembers
+public final class EmailSenderTool: NSObject, Tool {
+
+    // MARK: - Public properties -
+    
+    public var name: String { "Email Sender" }
```

Comment:
I would expose this through `init` method to give developers ability to change the title if needed.